ConsoleApp/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+System.Console.Write("Input first num(only numbers):");
+var num1 = Console.ReadLine();
+System.Console.Write("Input second num(only numbers):");
+var num2 = Console.ReadLine();
+
+if(num1 == num2) System.Console.WriteLine($"Numbers are equal {num1}"); //or System.Console.WriteLine($"Numbers are equal");
+
+else if (int.TryParse(num1, out int pNumber1) && int.TryParse(num2, out int pNumber2))
+{
+    GetSum(pNumber1,pNumber2);
+}
+
+else
+{
+    System.Console.WriteLine("You're stupid!");
+    System.Console.WriteLine("Press any button to exit the program...");
+    Console.ReadKey();
+    Environment.Exit(0);
+}
+
+
+void GetSum(int a, int b)
+{
+    int sum = 0;
+    for (int i = Math.Min(a,b); i <= Math.Max(a,b); ++i)
```

Comment:
here it's better to place the result of `Math.Max(a,b)` in dedicated variable because each time you check the condition - you execute the function